src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,23 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeModal();
+    }
+
+    public void closeModal() {
+        WebElement closeButton = browser.driver.findElement(By.className("close"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        try {
+            Thread.sleep(1000);
```

Comment:
@chowyb that is rather strange because I'm still getting the timeout exception on my local test. Committing the changes to see what Travis has to say.